Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM postgres:9.5.4
+
+ENV GO_VERSION 1.7.1
+ENV GO_HOST_ARCH amd64
+ENV GO_HOST_OS linux
+ENV GO_ROOT /usr/local/go
+ENV GO_DL_BASE https://storage.googleapis.com/golang
+ENV GO_DL_URL $GO_DL_BASE/go$GO_VERSION.$GO_HOST_OS-$GO_HOST_ARCH.tar.gz
```

Comment:
These env variables will be exported to the app but they are only needed to compose the `GO_DL_URL` (and are creating some confusion with the go ones). I'd avoid using env variables as shell variables just do something like:

```
curl  https://storage.googleapis.com/golang/go1.7.1.linux-amd64.tar.gz
```
since all that variables are fixed anyway.